docs/topics/api/addons.rst

```diff
@@ -76,6 +78,18 @@ This endpoint allows you to search through public add-ons.
     sort parameter, which is only available alone.
 
 
+.. _addon-threshold-param:
+
+    Threshold style parameters allow queries against numeric values using comparison.
+
+    The following is supported (examples for query parameter `foo`):
+    * greater than ``foo__gt`` (example query: ?foo__gt=10.1)
+    * less than ``foo__lt`` (example query: ?foo__lt=10.1)
+    * greater than or equal to ``foo__gte`` (example query: ?foo__gte=10.1)
+    * less than or equal to ``foo__lte`` (example query: ?foo__lte=10.1)
+    * equal to ``foo`` (example query: ?foo=10.1)
+
+
```

Comment:
This is missing the indentation to make a list, it would squish everything on a single line.

Reply:
https://github.com/mozilla/addons-server/pull/17602/commits/c202cf7f70a2874cf635f4d20c48054a43ebb75f